### ISSUE TITLE
Text without whitebackground remains black in darkmode

### ISSUE THREAD

**Issue by b21phiro** [HGustavs/LenaSYS#13869]
## Description
Text that does not have any backgrounds remains black in darkmode, making the text impossible to read.

Figure 1.
![bild](https://user-images.githubusercontent.com/45258330/237014457-b7bb07d4-3c04-4282-94c0-8f01c59c9a8c.png)

## To do
Make sure text without background becomes readable regardless of light/dark theme.

**Comment by b21phiro** [comment]
Sure, whatever makes the text visible.

**Comment by HGustavs** [comment]
For all dark mode handling I do think that we decided that inversion of colors (e.g. black to white, white to black) should be the default behavior for dark mode?

**Comment by b21phiro** [comment]
> For all dark mode handling I do think that we decided that inversion of colors (e.g. black to white, white to black) should be the default behavior for dark mode?

Yes, that is true. We could do just that the text inverts color.
